extensions/sql-database-projects/src/typings/vscode-mssql.d.ts

```diff
@@ -532,4 +555,25 @@ declare module 'vscode-mssql' {
 
         schema: string;
     }
+
+    export const enum BindingType {
```

Comment:
These should ideally have doc comments (both here and in vs code mssql)

Reply:
added more comments. I'll send out a PR to update it in vscode mssql too